lib/helpers.js

```diff
@@ -26,6 +26,17 @@ _beforeEach.withApp = function(app) {
   });
 }
 
+_beforeEach.cleanDatasource = function(dsName) {
+  beforeEach(function() {
+    if(!dsName) dsName = 'db';
+
+    if (this.app) {
+      this.app.datasources[dsName].automigrate();
```

Comment:
Ensure `automigrate()` completes... As we have it here it will run alongside the tests...